stories/assets/scss/_base.scss

```diff
@@ -330,27 +353,36 @@ $color in $colors {
       //feature-card
       .feature__card-slide {
         &::before {
-          @include mixins.background-image(318deg, #{$color}, transparent 67.25%);
+          @include mixins.background-image(
+            318deg,
+            #{$color},
+            transparent 67.25%
+          );
         }
       }
     }
   }
+
+  /* Custom offset class for RTL: adds right margin instead of left on medium screens */
+  @include breakpoints.devicebreak(medium) {
```

Comment:
This treatment should be applied to the existing markup without a necessity to add any new class. Actually, it can be applied to the .medium-offset-1 directly, so if that class is used inside container with [dir="rtl"] attribute - it will get proper right margin and not left. 
And please use margin-inline-start, margin-inline-end instead of margin-left and margin-right